src/Kovan-DssSpell.t.sol

```diff
@@ -246,10 +246,10 @@ contract DssSpellTest is DSTest, DSMath {
         // Test for spell-specific parameters
         //
         spellValues = SpellValues({
-            deployed_spell:                 address(0x8c38736FF2dA058a3aef53E59f39A42b34104636),        // populate with deployed spell if deployed
-            deployed_spell_created:         1628088292,        // use get-created-timestamp.sh if deployed
+            deployed_spell:                 address(0),        // populate with deployed spell if deployed
+            deployed_spell_created:         0,        // use get-created-timestamp.sh if deployed
             previous_spell:                 address(0),        // supply if there is a need to test prior to its cast() function being called on-chain.
-            office_hours_enabled:           false,              // true if officehours is expected to be enabled in the spell
+            office_hours_enabled:           true,              // true if officehours is expected to be enabled in the spell
```

Comment:
So we can disable officehours for the kovan deployment, but we should make a todo to change this on mainnet.

Reply:
done